src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,42 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final CarService carService
+            = (CarService) injector.getInstance(CarService.class);
+    private static final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
```

Comment:
```suggestion
    private final CarService carService
            = (CarService) injector.getInstance(CarService.class);
    private final DriverService driverService
            = (DriverService) injector.getInstance(DriverService.class);
```

Reply:
The previous mentor's comment was not fixed. Fix it before new review request
![image](https://user-images.githubusercontent.com/20326858/200548990-b66e9655-eb73-4984-b1d7-cf34e68d724d.png)